commit 7f0d9e61f40c669fca3cfd1e342fa8236c7220b7
Author: Paul Pluzhnikov <ppluzhnikov@google.com>
Date:   Sat May 20 13:37:47 2023 +0000

    Fix all the remaining misspellings -- BZ 25337

Note on the changes:
- Partial backport, the only file modified is `resolv/inet_pton.c` to
  ease further backport for RHEL-83984.

diff --git a/resolv/inet_pton.c b/resolv/inet_pton.c
index f1d5db75d0d47501..835f364794c1be96 100644
--- a/resolv/inet_pton.c
+++ b/resolv/inet_pton.c
@@ -121,7 +121,7 @@ inet_pton4 (const char *src, const char *end, unsigned char *dst)
   return 1;
 }
 
-/* Return the value of CH as a hexademical digit, or -1 if it is a
+/* Return the value of CH as a hexadecimal digit, or -1 if it is a
    different type of character.  */
 static int
 hex_digit_value (char ch)
